File: src/main/java/cz/craftmania/logger/listeners/CommandLogListener.java
package cz.craftmania.logger.listeners;

import cz.craftmania.logger.Main;
import cz.craftmania.logger.utils.Log;
import org.bukkit.event.EventHandler;
import org.bukkit.event.Listener;
import org.bukkit.event.player.PlayerCommandPreprocessEvent;

public class CommandLogListener implements Listener {

    @EventHandler(ignoreCancelled = true)
    public void onPlayerCommand(final PlayerCommandPreprocessEvent event){
        String[] split = event.getMessage().split(" ");
        for (int length = split.length, i = 0; i < length; ++i) {
            final String word = split[i];
            if (Main.getInstance().getConfig().getStringList("logger.commands.ignored").contains(word)) {
                return;
            }
        }
        Main.getInstance().getSQL().createCommandLog(event.getPlayer(), "COMMAND", event.getMessage(), System.currentTimeMillis());
        Log.debug("[A:COMMAND]: " + event.getPlayer().getName() + "(" + event.getPlayer().getUniqueId().toString() + ") -> " + event.getMessage());

    }
}

File: src/main/java/cz/craftmania/logger/listeners/CommandLogListener.kt
package cz.craftmania.logger.listeners

import cz.craftmania.logger.Main
import cz.craftmania.logger.utils.Log
import org.bukkit.event.EventHandler
import org.bukkit.event.Listener
import org.bukkit.event.player.PlayerCommandPreprocessEvent

class CommandLogListener : Listener {

    @EventHandler(ignoreCancelled = true)
    fun onPlayerCommand(event: PlayerCommandPreprocessEvent) {
        val split = event.message.split(" ").toTypedArray()
        val length = split.size
        var i = 0
        while (i < length) {
            val word = split[i]
            if (Main.instance!!.config.getStringList("logger.commands.ignored").contains(word)) {
                return
            }
            ++i
        }
        Main.instance!!.sQL!!.createDataLog(
            event.player,
            "command",
            "COMMAND",
            event.message,
            System.currentTimeMillis()
        )
        Log.debug("[A:COMMAND]: " + event.player.name + "(" + event.player.uniqueId.toString() + ") -> " + event.message)
    }
}